valhalla/baldr/edge_elevation.h

```diff
@@ -0,0 +1,84 @@
+#ifndef VALHALLA_BALDR_EDGEELEVATION_H_
+#define VALHALLA_BALDR_EDGEELEVATION_H_
+
+namespace valhalla {
+namespace baldr {
+
+// Use elevation bins of 8 meters to store mean elevation. Clamp to a range
+// from -500 meters to 7683 meters.
+constexpr uint32_t kMaxStoredElevation = 1023;
+constexpr float kElevationBinSize = 8.0f;
```

Comment:
@dnesbitt61 Dave, actually can we increase resolution to 1m (13 bits), in case we want to change elevation data source (with a higher precision one).

Alternatively - move `mean_elevation_` down next to spare bits, so it can be changed afterwards